imap/command.c

```diff
@@ -845,91 +853,79 @@ static void cmd_parse_status(struct ImapAccountData *adata, char *s)
 
     if (mutt_str_startswith(s, "MESSAGES", CASE_MATCH))
     {
-      status->messages = count;
+      mdata->messages = count;
       new_msg_count = 1;
     }
     else if (mutt_str_startswith(s, "RECENT", CASE_MATCH))
-      status->recent = count;
+      mdata->recent = count;
     else if (mutt_str_startswith(s, "UIDNEXT", CASE_MATCH))
-      status->uidnext = count;
+      mdata->uid_next = count;
     else if (mutt_str_startswith(s, "UIDVALIDITY", CASE_MATCH))
-      status->uidvalidity = count;
+      mdata->uid_validity = count;
     else if (mutt_str_startswith(s, "UNSEEN", CASE_MATCH))
-      status->unseen = count;
+      mdata->unseen = count;
 
     s = value;
     if (*s && *s != ')')
       s = imap_next_word(s);
   }
   mutt_debug(3, "%s (UIDVALIDITY: %u, UIDNEXT: %u) %d messages, %d recent, %d unseen\n",
-             status->name, status->uidvalidity, status->uidnext,
-             status->messages, status->recent, status->unseen);
+             mdata->name, mdata->uid_validity, mdata->uid_next, mdata->messages,
+             mdata->recent, mdata->unseen);
 
+#if 0
+  // NOTE(sileht): I don't get this part, I haven't found where cmddata is reused
+  // after that.
   /* caller is prepared to handle the result herself */
   if (adata->cmddata && adata->cmdtype == IMAP_CT_STATUS)
   {
     memcpy(adata->cmddata, status, sizeof(struct ImapStatus));
     return;
   }
+#endif
```

Comment:
I agree, this block can go (and the `IMAP_CT_STATUS` enum value).
It isn't used and I can't find any evidence it was _ever_ used for 'STATUS'.

This means we don't need `enum ImapCommandType` any more
We might want to rename `cmddata` as it's only used by 'LIST'.